proto/tendermint/types/evidence.proto

```diff
@@ -12,13 +13,18 @@ import "tendermint/crypto/keys.proto";
 message DuplicateVoteEvidence {
   Vote vote_a = 1;
   Vote vote_b = 2;
+  
+  google.protobuf.Timestamp time = 3
```

Comment:
Why are these `timestamp` in the Go structs but `time` in the protobufs?

Reply:
I renamed the go structs to Timestamp because Time is already taken by the interface

Reply:
I can also rename the protobufs to keep them consistent